[2024-11-28T10:16:47.556+0700] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T10:16:47.562+0700] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: rocket_launch_pipeline.extract_images manual__2024-11-06T11:15:45.250198+00:00 [queued]>
[2024-11-28T10:16:47.565+0700] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: rocket_launch_pipeline.extract_images manual__2024-11-06T11:15:45.250198+00:00 [queued]>
[2024-11-28T10:16:47.565+0700] {taskinstance.py:2865} INFO - Starting attempt 4 of 4
[2024-11-28T10:16:47.573+0700] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_images> on 2024-11-06 11:15:45.250198+00:00
[2024-11-28T10:16:47.575+0700] {standard_task_runner.py:72} INFO - Started process 81976 to run task
[2024-11-28T10:16:47.578+0700] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'rocket_launch_pipeline', 'extract_images', 'manual__2024-11-06T11:15:45.250198+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpyxo_j3qi']
[2024-11-28T10:16:47.578+0700] {standard_task_runner.py:105} INFO - Job 137: Subtask extract_images
[2024-11-28T10:16:47.602+0700] {task_command.py:467} INFO - Running <TaskInstance: rocket_launch_pipeline.extract_images manual__2024-11-06T11:15:45.250198+00:00 [running]> on host Web-container
[2024-11-28T10:16:47.634+0700] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='rocket_launch_pipeline' AIRFLOW_CTX_TASK_ID='extract_images' AIRFLOW_CTX_EXECUTION_DATE='2024-11-06T11:15:45.250198+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-06T11:15:45.250198+00:00'
[2024-11-28T10:16:47.634+0700] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-28T10:16:47.634+0700] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-28T10:16:47.634+0700] {logging_mixin.py:190} INFO - Current task name:extract_images state:running start_date:2024-11-28 03:16:47.562684+00:00
[2024-11-28T10:16:47.635+0700] {logging_mixin.py:190} INFO - Dag name:rocket_launch_pipeline and current dag run status:running
[2024-11-28T10:16:47.635+0700] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-28T10:16:57.603+0700] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-28T10:16:57.618+0700] {process_utils.py:132} INFO - Sending Signals.SIGTERM to group 81976. PIDs of all processes in the group: [81976]
[2024-11-28T10:16:57.619+0700] {process_utils.py:87} INFO - Sending the signal Signals.SIGTERM to group 81976
[2024-11-28T10:16:57.619+0700] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-28T10:16:57.626+0700] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/dags/dag.py", line 37, in extract_images
    download_image(url)
  File "/home/fawzan/airflow/dags/dag.py", line 40, in download_image
    response = requests.get(url)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 789, in urlopen
    response = self._make_request(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 536, in _make_request
    response = conn.getresponse()
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/urllib3/connection.py", line 464, in getresponse
    httplib_response = super().getresponse()
  File "/usr/lib/python3.10/http/client.py", line 1375, in getresponse
    response.begin()
  File "/usr/lib/python3.10/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.10/http/client.py", line 279, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.10/ssl.py", line 1303, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.10/ssl.py", line 1159, in read
    return self._sslobj.read(len, buffer)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3094, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-11-28T10:16:57.659+0700] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-11-28T10:16:57.660+0700] {logging_mixin.py:190} INFO - Task start:2024-11-28 03:16:47.562684+00:00 end:2024-11-28 03:16:57.658635+00:00 duration:10.095951
[2024-11-28T10:16:57.660+0700] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): extract_images> dag:<DAG: rocket_launch_pipeline> dagrun:<DagRun rocket_launch_pipeline @ 2024-11-06 11:15:45.250198+00:00: manual__2024-11-06T11:15:45.250198+00:00, state:running, queued_at: 2024-11-06 11:15:45.271017+00:00. externally triggered: True>
[2024-11-28T10:16:57.660+0700] {logging_mixin.py:190} INFO - Failure caused by Task received SIGTERM signal
[2024-11-28T10:16:57.661+0700] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=rocket_launch_pipeline, task_id=extract_images, run_id=manual__2024-11-06T11:15:45.250198+00:00, execution_date=20241106T111545, start_date=20241128T031647, end_date=20241128T031657
[2024-11-28T10:16:57.673+0700] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-28T10:16:57.711+0700] {process_utils.py:80} INFO - Process psutil.Process(pid=81976, status='terminated', exitcode=2, started='10:16:46') (81976) terminated with exit code 2
[2024-11-28T10:16:59.653+0700] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T10:16:59.656+0700] {taskinstance.py:2602} INFO - Dependencies not met for <TaskInstance: rocket_launch_pipeline.extract_images manual__2024-11-06T11:15:45.250198+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2024-11-28T10:16:59.659+0700] {local_task_job_runner.py:166} INFO - Task is not able to be run
